ui-tests/src/main/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -76,8 +76,8 @@
 	private static final By PATIENT = By.xpath("//ul[@id='breadcrumbs']/li[2]/a");
 
 	private static final By EYE_REPORT = By.linkText("Eye Report");
-	private static final By TELEPHONE_NUMBER_TEXT = By.cssSelector("#contactInfoContent div span.left-margin");
 	private static final By RECENT_VISITS = By.cssSelector("#content div.container  div  div:nth-child(2)  div:nth-child(1)  div.info-header  a");

```

Comment:
It seems like a very unreliable selector. Could you please add an id to the element?